### ISSUE TITLE
dex/testing/dcr: simnet harness: beta node hangs mining second block after disconnecting from alpha

### ISSUE THREAD

**Issue by buck54321** [decred/dcrdex#646]
This behavior breaks the reorg script.

To repro, you can just run the `./reorg` script in the simnet harness. It might be more informative to do it manually though. 
1. `./beta addnode 127.0.0.1:19571 remove`
2. `./mine-beta 1`
3. `./mine-beta 1`

I do see a debug-level message in the beta nodes logs that says `Too few voters found on any HEAD block, recycling a parent block to mine on`. I tried to set `ENABLE_TICKET_BUYER="1"` for the beta wallet, but to no avail.

**Comment by davecgh** [comment]
If you look at how I did this in the dcrd simnet harness, I create a second wallet that is connected to the second dcrd node for this reason.  When the second node is disconnected from the first, it has no way to receive votes if you don't also have a voting wallet connected to it to provide them.

**Comment by buck54321** [comment]
I actually do have a wallet connected to that node, and setting `ENABLE_TICKET_BUYER="1"` adds this to the wallet config.

```
enablevoting=1
enableticketbuyer=1
ticketbuyer.limit=6
ticketbuyer.balancetomaintainabsolute=50
```

I lowered `balancetomaintainabsolute` to 50 since the beta wallet had only ~ 80 in spendable balance. Stake diff is only 0.03 or so, but I'm still not getting it. Hmmm.

**Comment by davecgh** [comment]
There really should only be a single wallet with the ticketbuyer enabled and they both should be on the same seed.  Otherwise, the other wallet might have the tickets and the second one can't vote.

**Comment by chappjc** [comment]
Could there be no < 30dcr outpoint?

**Comment by itswisdomagain** [comment]
I recently encountered this issue and as @davecgh said, only 1 block mined on the second node will extend the mainchain, any other blocks generated will create sidechains at the same height. I did something to smoothen out the reorg script while testing locally and could PR it in if something else isn't in the works already. Had issues with the btc harness reorg script as well that required some tweaking locally.
